app/components/alerts-pre.jsx

```diff
@@ -0,0 +1,11 @@
+var React = require("React");
+
+let Alerts = React.createClass({
```

Comment:
Remove the `s` from each of the class names. The class will only give you 1 Alert element, not multiple.